proto/osmosis/gamm/pool-models/stableswap/stableswap_pool.proto

```diff
@@ -63,4 +63,9 @@ message Pool {
     (gogoproto.nullable) = false,
     (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins"
   ];
+  // for calculation amognst assets with different precisions
+  repeated uint64 scalingFactor = 7 [
```

Comment:
```suggestion
  repeated uint64 scaling_factor = 7 [
```

Reply:
seems that we have mixed use cases of camel cases and snake cases within Pool struct. Which styling should we try to follow?